metricbeat-yml-script.py

```diff
@@ -179,6 +179,21 @@ def _add_aws_shipping_data():
             logger.error("Could not find aws access key or secret key or region: {}".format(KeyError))
 
 
+def _get_tags_value(aws_namespace):
+    service_name = aws_namespace.split("/")[1].lower()
+    tags_val = service_name
+    if service_name == "ebs":
+        tags_val = "ec2:snapshot"
+    if service_name == "elb" or service_name == "applicationelb" or service_name == "networkelb":
+        tags_val = "elasticloadbalancing"
+    if service_name == "amazonmq":
+        tags_val = "mq"
+    if service_name == "efs":
+        tags_val = "elasticfilesystem"
+
+    return tags_val
+
+
```

Comment:
Is it possible to have a few tags from different services in this function? also, what will happen if tags_val = service_name?

Reply:
The function is being called during an iteration on the chosen services, so we only need one service for every call.
For most of the services, the tags_val is usually the AWS namespace in lowercase. Only in some services (those that are in "if" statements- elb, amazonmq, etc...) it's not like that. So the default is tags_val = service_name, unless it's one of the mentioned services.

Reply:
@mirii1994 What will happen if we use AWS namespace as tags_val and it's not a valid value for metricbeat? will metricbeat fail? 

If only one value is accepted I will use if/elif, drop tags_val and use return on each statement, and return service_name at the end of the function

Reply:
When the value of the `tags.resource_type_filter` is invalid it logs the following error, but continues working and sending metrics: `getResourcesTags failed, skipping region <region name>: error GetResources: InvalidParameterException: Unsupported service=<invalid_value> in ResourceTypeFilter - <invalid_value>`.

I'll change the function to be in if/elif structure you suggested.

Reply:
@mirii1994 Are we sure it keeps sending the metrics from the services it failed on?

Reply:
Yes. Also, made the structure change in the tags function :)